.buildkite/scripts/win_unit_tests.ps1

```diff
@@ -120,6 +124,14 @@ function withNmap($version) {
     }
     Start-Process -FilePath $nmapDownloadPath -ArgumentList "/S" -Wait
 }
+function google_cloud_auth {
+    $tempFileName = "google-cloud-credentials.json"
+    $secretFileLocation = Join-Path $env:TEMP $tempFileName
```

Comment:
Shall this file be deleted after being used?

Reply:
This discussion has not been answered

Reply:
I've added cleanup [here](https://github.com/elastic/beats/pull/38135/commits/d02580b612334c9b2875a1d3c3a8bda71216ba83), [there is](https://buildkite.com/elastic/beats-xpack-packetbeat/builds/796#018e378a-9aef-49ac-b056-6328a011d99f/101-309) the result, please retake a look.